packages/fxa-auth-server/scripts/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -0,0 +1,328 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import program from 'commander';
+import Stripe from 'stripe';
+import Container from 'typedi';
+
+import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-setup';
+import { AppConfig, AuthFirestore } from '../lib/types';
+
+const pckg = require('../package.json');
+
+const MONTHLY_NOTICE_DAYS = 14;
+const YEARLY_NOTICE_DAYS = 30;
+const SECONDS_TO_MILLISECONDS = 1000;
+
+// Represents a mapping of user uids to IP addresses
+interface IpAddressMap {
+  [key: string]: string;
+}
+
+// Firestore subscriptions contain additional expanded information
+// on top of the base Stripe.Subscription type
+interface FirestoreSubscription extends Stripe.Subscription {
+  customer: string;
+  plan: Stripe.Plan;
+  price: Stripe.Price;
+}
+
+const parseDryRun = (dryRun: boolean | string) => {
+  return `${dryRun}`.toLowerCase() !== 'false';
+};
+
+const parseBatchSize = (batchSize: string | number) => {
+  return parseInt(batchSize.toString(), 10);
+};
+
+const parseOutputFile = (outputFile: string) => {
+  return outputFile;
+};
+
+const isStripeTaxDisabled = (subscription: Stripe.Subscription) => {
+  return !subscription.automatic_tax.enabled;
+};
+
+// Checks whether subscription renews too soon to be upgraded to
+// Stripe Automatic Tax
+// 1 month plans require at least 14 days of notice
+// 6 month and yearly plans require at least 30 days of notice
+// This function does not support plans with intervals of less than 1 month
+const isWithinNoticePeriod = (sub: Stripe.Subscription) => {
+  const noSoonerThan = new Date();
+  if (sub.items.data[0].plan.interval === 'month') {
+    noSoonerThan.setUTCDate(noSoonerThan.getUTCDate() + MONTHLY_NOTICE_DAYS);
+  } else {
+    noSoonerThan.setUTCDate(noSoonerThan.getUTCDate() + YEARLY_NOTICE_DAYS);
+  }
+
+  const renewalDate = new Date(
+    sub.current_period_end * SECONDS_TO_MILLISECONDS
```

Comment:
since `SECONDS_TO_MILLISECONDS` is only used in one place, why not just hardcode the `1000` here?